packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -342,6 +349,33 @@ const ComboBox = forwardRef(
       slug,
       ...rest
     } = props;
+    const { refs, floatingStyles } = useFloating(
+      autoAlign
+        ? {
+            strategy: 'fixed',
```

Comment:
I think if we include the `placement` config option with the `direction` consumers will still be able to specify it to open `top` by default

```suggestion
            placement: direction,
            strategy: 'fixed',
```